### DISCUSSION TITLE
Sampler1D disappers when a new node point is created with shift+right click

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#228] [Feedback & Feature Requests]
If I shift right click a wire to create a node point, sampler1D will disappear.  

It is not a urgent problem though. 

![Bug](https://user-images.githubusercontent.com/36077492/151710273-4944b134-c805-4614-80b8-7d0e018a74ec.JPG)

**Comment by pragma37** [comment]
Should be fixed now.
Please, for future bug reports open an issue instead:
https://github.com/bnpr/Malt/issues/new/choose

**Reply by HSDSZ** [reply]
noted. I will pay attention next time
